src/tiled/tilesetview.cpp

```diff
@@ -814,6 +814,9 @@ void TilesetView::contextMenuEvent(QContextMenuEvent *event)
     connect(toggleGrid, &QAction::toggled,
             prefs, &Preferences::setShowTilesetGrid);
 
+    QAction *selectAllTiles = menu.addAction(tr("Select &All"));
```

Comment:
I'd prefer "Select &All Tiles" here, to make this action a little more specific.

Reply:
@bjorn done!